governance/handler.go

```diff
@@ -459,3 +459,12 @@ func (gov *Governance) GetGoverningInfoAtNumber(num uint64) (bool, common.Addres
 
 	return true, govNode.(common.Address), nil
 }
+
+func (gov *Governance) GetMinimumStakingAtNumber(num uint64) (uint64, error) {
+	minStaking, err := gov.GetItemAtNumberByIntKey(num, params.MinimumStake)
+	if err != nil {
+		return 0, err
+	}
+	bigMinStaking, _ := new(big.Int).SetString(minStaking.(string), 10)
+	return bigMinStaking.Uint64(), nil
```

Comment:
```suggestion
	if bigMinStaking, ok := new(big.Int).SetString(minStaking.(string), 10); ok {
		return bigMinStaking.Uint64(), nil	
	}
```

How about checking if func SetString returns true?

Reply:
Reflected. I have added an error case.
Thanks :)